internal/bufferedwrites/upload_handler_test.go

```diff
@@ -67,6 +71,90 @@ func (t *UploadHandlerTest) SetupSubTest() {
 	t.SetupTest()
 }
 
+func (t *UploadHandlerTest) createUploadHandlerWithNonNilObjectOfGivenSize(size uint64, finalized time.Time) {
```

Comment:
can you rename this to createUploadHandlerWithObjectOfGivenSize
the object is nonnil always